app/assets/stylesheets/tag.scss

```diff
@@ -23,11 +23,13 @@ h1.tag {
   }
 }
 .page-tags {
-  background-color: $sidebars-background;
-  .left-navbar-fixed-background { border-right: 0; }
   #tags_show {
-    .col-md-3 {
-      h4 { margin: 25px 0 15px; }
+    .sidebar {
+      h3 {
```

Comment:
I'll remove the `.page-tags` to reduce the depth in a following PR: it modifies every lines so pronto is crying about ids used etc, I don't want to do any more noise here.